Description: Hard-code paths to external system run-time dependencies
Author: Tobias Hansen <thansen@debian.org>
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/interfaces/gap.py
+++ b/sage/src/sage/interfaces/gap.py
@@ -197,8 +197,7 @@
 
 WORKSPACE = gap_workspace_file()
 
-from sage.env import GAP_ROOT_DIR
-GAP_BINARY = os.path.join(GAP_ROOT_DIR, 'bin', 'gap.sh')
+GAP_BINARY = '/usr/bin/gap'
 
 first_try = True
 
--- a/sage/src/sage/misc/cython.py
+++ b/sage/src/sage/misc/cython.py
@@ -582,7 +582,7 @@
     extra_sources = []
     for fname in additional_source_files:
         fname = fname.replace("$SAGE_SRC", SAGE_SRC)
-        fname = fname.replace("$SAGE_LOCAL", SAGE_LOCAL)
+        fname = fname.replace("$SAGE_LOCAL", '/usr/include')
         extra_sources.append(fname)
 
     # Add current working directory to includes. This is needed because
--- a/sage/src/sage/libs/lcalc/lcalc_sage.h
+++ b/sage/src/sage/libs/lcalc/lcalc_sage.h
@@ -1,4 +1,4 @@
-#include "libLfunction/L.h"
+#include "Lfunction/L.h"
 
 int *new_ints(int l)
 {
--- a/sage/build/bin/sage-python23
+++ b/sage/build/bin/sage-python23
@@ -12,9 +12,9 @@
 
 
 if [ "$SAGE_PYTHON3" = yes ]; then
-   PYTHON="$SAGE_LOCAL/bin/python3"
+   PYTHON="/usr/bin/python3"
 else
-   PYTHON="$SAGE_LOCAL/bin/python2"
+   PYTHON="/usr/bin/python2"
 fi
 
 # Check that Python is actually installed and issue an error message if not--in
